actionManifest.json

```diff
@@ -0,0 +1,55 @@
+{
+    "tag": "2048-teams-app",
```

Comment:
Update this as per : https://github.com/OfficeDev/microsoft-teams-apps-survey/blob/main/actionManifest.json

Reply:
please validate the whole action Manifest schema as per the shared link